lib/Serialization/Deserialization.cpp

```diff
@@ -4153,7 +4153,7 @@ void ModuleFile::finishNormalConformance(NormalProtocolConformance *conformance,
       // Create an archetype builder, which will help us create the
       // synthetic environment.
       ArchetypeBuilder builder(*getAssociatedModule());
-      builder.addGenericSignature(syntheticSig, nullptr);
+      builder.addGenericSignature(syntheticSig, nullptr, /* FIXME */ nullptr);
```

Comment:
This is deserializing a conformance, and I'm not sure if there's any `Decl` that makes sense here?